home/models.py

```diff
@@ -1268,3 +1269,251 @@ class PageView(models.Model):
     )
     message = models.IntegerField(blank=True, default=None, null=True)
     data = models.JSONField(default=dict, blank=True, null=True)
+
+
+class TemplateContentQuickReply(TagBase):
+
+    class Meta:
+        verbose_name = "quick reply2"
+        verbose_name_plural = "quick replie2"
```

Comment:
Is there a reason for this naming, I don't think that `verbose_name` is used as an identifier anywhere, it's just what's shown on the front end

Reply:
Nah this was me testing something and forgot to change it back.  Updated